### PR TITLE
Add movielens example

### PR DESCRIPTION
**Pull request by grasskin**
Jax is still ahead on loss the trend is very exciting

### CODE REVIEW

File: examples/structured_data/md/collaborative_filtering_movielens.md [hunk 1]
```diff
@@ -166,20 +176,26 @@ class RecommenderNet(keras.Model):
...
 model.compile(
-    loss=tf.keras.losses.BinaryCrossentropy(),
+    loss=keras.losses.BinaryCrossentropy(),
     optimizer=keras.optimizers.Adam(learning_rate=0.001),
 )
 ```
 
+<div class="k-default-codeblock">
+```
+An NVIDIA GPU may be present on this machine, but a CUDA-enabled jaxlib is not installed. Falling back to cpu.
```
Comment (fchollet) [inline examples/structured_data/md/collaborative_filtering_movielens.md:195]: This is a big problem! You may want to fix that.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

Because the step time is already very low there's not need to rerun on GPU, but please manually remove the warning in the md file.

File: examples/structured_data/md/collaborative_filtering_movielens.md (hunk 1 above)
Comment (grasskin) [reply examples/structured_data/md/collaborative_filtering_movielens.md:195]: Reinstalled conda so likely have to redo some gpu things, thanks for finding! (will remove)

Comment (grasskin) [reply examples/structured_data/md/collaborative_filtering_movielens.md]: Fixed!

**Review by fchollet — APPROVED** [review]
LGTM
